policyengine_uk/parameters/calibration/programs/private_school_vat/private_school_fees.yaml

```diff
@@ -0,0 +1,8 @@
+description: Mean annual private school fee
+values:
+  2022-01-01: 15_200
+metadata:
+  unit: currency-GBP
```

Comment:
We'll want to add a projection onto future years here.

Reply:
Could you describe that a bit more? The source of this number only gave a number for the 2022/23 year, hence it being put in like that